package.json

```diff
@@ -40,6 +40,7 @@
     "@types/lodash": "^4.14.159",
     "@types/uuid": "^9.0.4",
     "apexcharts": "^3.26.2",
+    "axios": "^1.5.1",
```

Comment:
Using axios is normally a good idea - it's a great library, however we already have a service handling API calls: see /src/app/shared/infrastructure/api/callApi.service.ts
For an example of its use see: /src/app/welcome/pages/welcomeUseCase.ts